app/services/GovSpendService.scala

```diff
@@ -24,6 +24,7 @@ object GovSpendService {
     case 2016 => taxYear2016
     case 2017 => taxYear2017
     case 2018 => taxYear2018
+    case 2019 => taxYear2019
```

Comment:
This will throw a match error if the year is outside the range. We should with return something more meaningful like an `Option` or an `Either` or throw an exception with more detail so it is easier to understand what is going on in the logs